Kokoro tests (including nightly builds) are failing

Some tests start to fail today, and this was reproduced in #3552. Running the test once more now to check if this is still reproducible.

<details>
<summary>CloudLibrariesInPluginXmlTest.testObjectifyLibraryConfig and testObjectify6LibraryConfig</summary>

```
00:33:42.535 [main] ERROR c.g.c.t.e.a.libraries.model.Library - Could not load library MavenCoordinates [repository=central, com.googlecode.objectify:objectify:jar:null:6.0.5]
org.eclipse.core.runtime.CoreException: Could not resolve dependencies
	at com.google.cloud.tools.eclipse.util.DependencyResolver._getTransitiveDependencies(DependencyResolver.java:103) ~[com.google.cloud.tools.eclipse.util-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.util.DependencyResolver.lambda$0(DependencyResolver.java:63) ~[com.google.cloud.tools.eclipse.util-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.util.MavenUtils.lambda$1(MavenUtils.java:124) ~[com.google.cloud.tools.eclipse.util-0.1.0-SNAPSHOT.jar:na]
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.executeBare(MavenExecutionContext.java:177) ~[org.eclipse.m2e.core-1.9.0.20180606-2036.jar:na]
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.execute(MavenExecutionContext.java:151) ~[org.eclipse.m2e.core-1.9.0.20180606-2036.jar:na]
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.execute(MavenExecutionContext.java:99) ~[org.eclipse.m2e.core-1.9.0.20180606-2036.jar:na]
	at com.google.cloud.tools.eclipse.util.MavenUtils.runOperation(MavenUtils.java:120) ~[com.google.cloud.tools.eclipse.util-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.util.DependencyResolver.getTransitiveDependencies(DependencyResolver.java:60) ~[com.google.cloud.tools.eclipse.util-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryFactory.loadTransitiveDependencies(LibraryFactory.java:120) ~[com.google.cloud.tools.eclipse.appengine.libraries-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.appengine.libraries.model.Library.resolveDependencies(Library.java:211) [com.google.cloud.tools.eclipse.appengine.libraries-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.appengine.libraries.model.Library.getAllDependencies(Library.java:117) [com.google.cloud.tools.eclipse.appengine.libraries-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.appengine.libraries.model.CloudLibrariesInPluginXmlTest.testObjectifyLibraryConfig(CloudLibrariesInPluginXmlTest.java:151) [classes/:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_201]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_201]
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50) [junit.jar:4.12]
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12) [junit.jar:4.12]
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47) [junit.jar:4.12]
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17) [junit.jar:4.12]
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325) [junit.jar:4.12]
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78) [junit.jar:4.12]
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57) [junit.jar:4.12]
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290) [junit.jar:4.12]
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71) [junit.jar:4.12]
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288) [junit.jar:4.12]
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58) [junit.jar:4.12]
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268) [junit.jar:4.12]
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363) [junit.jar:4.12]
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365) [org.eclipse.tycho.surefire.junit4-1.3.0.jar:na]
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273) [org.eclipse.tycho.surefire.junit4-1.3.0.jar:na]
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238) [org.eclipse.tycho.surefire.junit4-1.3.0.jar:na]
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159) [org.eclipse.tycho.surefire.junit4-1.3.0.jar:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_201]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_201]
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray2(ReflectionUtils.java:206) [surefire-api-2.22.0.jar:2.22.0]
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:161) [surefire-booter-2.22.0.jar:2.22.0]
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:84) [surefire-booter-2.22.0.jar:2.22.0]
	at org.eclipse.tycho.surefire.osgibooter.OsgiSurefireBooter.run(OsgiSurefireBooter.java:110) [org.eclipse.tycho.surefire.osgibooter-1.3.0.jar:na]
	at org.eclipse.tycho.surefire.osgibooter.HeadlessTestApplication.run(HeadlessTestApplication.java:21) [org.eclipse.tycho.surefire.osgibooter-1.3.0.jar:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_201]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_201]
	at org.eclipse.equinox.internal.app.EclipseAppContainer.callMethodWithException(EclipseAppContainer.java:587) [org.eclipse.equinox.app-1.3.500.v20171221-2204.jar:na]
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:198) [org.eclipse.equinox.app-1.3.500.v20171221-2204.jar:na]
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134) [org.eclipse.osgi-3.13.0.v20180409-1500.jar:na]
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104) [org.eclipse.osgi-3.13.0.v20180409-1500.jar:na]
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388) [org.eclipse.osgi-3.13.0.v20180409-1500.jar:na]
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243) [org.eclipse.osgi-3.13.0.v20180409-1500.jar:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_201]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_201]
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:656) [org.eclipse.equinox.launcher-1.5.0.v20180512-1130.jar:na]
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:592) [org.eclipse.equinox.launcher-1.5.0.v20180512-1130.jar:na]
	at org.eclipse.equinox.launcher.Main.run(Main.java:1498) [org.eclipse.equinox.launcher-1.5.0.v20180512-1130.jar:na]
	at org.eclipse.equinox.launcher.Main.main(Main.java:1471) [org.eclipse.equinox.launcher-1.5.0.v20180512-1130.jar:na]
Caused by: org.eclipse.aether.resolution.DependencyResolutionException: Failed to collect dependencies at com.googlecode.objectify:objectify:jar:6.0.5 -> com.google.cloud:google-cloud-datastore:jar:1.101.0 -> io.grpc:grpc-core:jar:1.24.1 -> io.grpc:grpc-api:jar:[1.24.1]
	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveDependencies(DefaultRepositorySystem.java:351) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at com.google.cloud.tools.eclipse.util.DependencyResolver._getTransitiveDependencies(DependencyResolver.java:92) ~[com.google.cloud.tools.eclipse.util-0.1.0-SNAPSHOT.jar:na]
	... 59 common frames omitted
Caused by: org.eclipse.aether.collection.DependencyCollectionException: Failed to collect dependencies at com.googlecode.objectify:objectify:jar:6.0.5 -> com.google.cloud:google-cloud-datastore:jar:1.101.0 -> io.grpc:grpc-core:jar:1.24.1 -> io.grpc:grpc-api:jar:[1.24.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.collectDependencies(DefaultDependencyCollector.java:293) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveDependencies(DefaultRepositorySystem.java:307) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	... 60 common frames omitted
Caused by: org.eclipse.aether.resolution.VersionRangeResolutionException: No versions available for io.grpc:grpc-api:jar:[1.24.1] within specified range
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.filterVersions(DefaultDependencyCollector.java:651) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.processDependency(DefaultDependencyCollector.java:396) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.processDependency(DefaultDependencyCollector.java:365) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.process(DefaultDependencyCollector.java:353) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.doRecurse(DefaultDependencyCollector.java:507) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.processDependency(DefaultDependencyCollector.java:460) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.processDependency(DefaultDependencyCollector.java:365) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.process(DefaultDependencyCollector.java:353) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.doRecurse(DefaultDependencyCollector.java:507) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.processDependency(DefaultDependencyCollector.java:460) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.processDependency(DefaultDependencyCollector.java:365) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.process(DefaultDependencyCollector.java:353) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.collectDependencies(DefaultDependencyCollector.java:256) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	... 61 common frames omitted
00:33:42.699 [main] ERROR c.g.c.t.e.a.libraries.model.Library - Could not load library MavenCoordinates [repository=central, com.googlecode.objectify:objectify:jar:null:6.0.5]
org.eclipse.core.runtime.CoreException: Could not resolve dependencies
	at com.google.cloud.tools.eclipse.util.DependencyResolver._getTransitiveDependencies(DependencyResolver.java:103) ~[com.google.cloud.tools.eclipse.util-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.util.DependencyResolver.lambda$0(DependencyResolver.java:63) ~[com.google.cloud.tools.eclipse.util-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.util.MavenUtils.lambda$1(MavenUtils.java:124) ~[com.google.cloud.tools.eclipse.util-0.1.0-SNAPSHOT.jar:na]
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.executeBare(MavenExecutionContext.java:177) ~[org.eclipse.m2e.core-1.9.0.20180606-2036.jar:na]
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.execute(MavenExecutionContext.java:151) ~[org.eclipse.m2e.core-1.9.0.20180606-2036.jar:na]
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.execute(MavenExecutionContext.java:99) ~[org.eclipse.m2e.core-1.9.0.20180606-2036.jar:na]
	at com.google.cloud.tools.eclipse.util.MavenUtils.runOperation(MavenUtils.java:120) ~[com.google.cloud.tools.eclipse.util-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.util.DependencyResolver.getTransitiveDependencies(DependencyResolver.java:60) ~[com.google.cloud.tools.eclipse.util-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryFactory.loadTransitiveDependencies(LibraryFactory.java:120) ~[com.google.cloud.tools.eclipse.appengine.libraries-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.appengine.libraries.model.Library.resolveDependencies(Library.java:211) [com.google.cloud.tools.eclipse.appengine.libraries-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.appengine.libraries.model.Library.getAllDependencies(Library.java:117) [com.google.cloud.tools.eclipse.appengine.libraries-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.appengine.libraries.model.CloudLibrariesInPluginXmlTest.testObjectify6LibraryConfig(CloudLibrariesInPluginXmlTest.java:206) [classes/:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_201]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_201]
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50) [junit.jar:4.12]
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12) [junit.jar:4.12]
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47) [junit.jar:4.12]
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17) [junit.jar:4.12]
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325) [junit.jar:4.12]
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78) [junit.jar:4.12]
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57) [junit.jar:4.12]
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290) [junit.jar:4.12]
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71) [junit.jar:4.12]
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288) [junit.jar:4.12]
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58) [junit.jar:4.12]
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268) [junit.jar:4.12]
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363) [junit.jar:4.12]
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365) [org.eclipse.tycho.surefire.junit4-1.3.0.jar:na]
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273) [org.eclipse.tycho.surefire.junit4-1.3.0.jar:na]
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238) [org.eclipse.tycho.surefire.junit4-1.3.0.jar:na]
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159) [org.eclipse.tycho.surefire.junit4-1.3.0.jar:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_201]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_201]
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray2(ReflectionUtils.java:206) [surefire-api-2.22.0.jar:2.22.0]
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:161) [surefire-booter-2.22.0.jar:2.22.0]
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:84) [surefire-booter-2.22.0.jar:2.22.0]
	at org.eclipse.tycho.surefire.osgibooter.OsgiSurefireBooter.run(OsgiSurefireBooter.java:110) [org.eclipse.tycho.surefire.osgibooter-1.3.0.jar:na]
	at org.eclipse.tycho.surefire.osgibooter.HeadlessTestApplication.run(HeadlessTestApplication.java:21) [org.eclipse.tycho.surefire.osgibooter-1.3.0.jar:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_201]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_201]
	at org.eclipse.equinox.internal.app.EclipseAppContainer.callMethodWithException(EclipseAppContainer.java:587) [org.eclipse.equinox.app-1.3.500.v20171221-2204.jar:na]
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:198) [org.eclipse.equinox.app-1.3.500.v20171221-2204.jar:na]
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134) [org.eclipse.osgi-3.13.0.v20180409-1500.jar:na]
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104) [org.eclipse.osgi-3.13.0.v20180409-1500.jar:na]
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388) [org.eclipse.osgi-3.13.0.v20180409-1500.jar:na]
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243) [org.eclipse.osgi-3.13.0.v20180409-1500.jar:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_201]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_201]
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:656) [org.eclipse.equinox.launcher-1.5.0.v20180512-1130.jar:na]
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:592) [org.eclipse.equinox.launcher-1.5.0.v20180512-1130.jar:na]
	at org.eclipse.equinox.launcher.Main.run(Main.java:1498) [org.eclipse.equinox.launcher-1.5.0.v20180512-1130.jar:na]
	at org.eclipse.equinox.launcher.Main.main(Main.java:1471) [org.eclipse.equinox.launcher-1.5.0.v20180512-1130.jar:na]
Caused by: org.eclipse.aether.resolution.DependencyResolutionException: Failed to collect dependencies at com.googlecode.objectify:objectify:jar:6.0.5 -> com.google.cloud:google-cloud-datastore:jar:1.101.0 -> io.grpc:grpc-core:jar:1.24.1 -> io.grpc:grpc-api:jar:[1.24.1]
	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveDependencies(DefaultRepositorySystem.java:351) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at com.google.cloud.tools.eclipse.util.DependencyResolver._getTransitiveDependencies(DependencyResolver.java:92) ~[com.google.cloud.tools.eclipse.util-0.1.0-SNAPSHOT.jar:na]
	... 59 common frames omitted
Caused by: org.eclipse.aether.collection.DependencyCollectionException: Failed to collect dependencies at com.googlecode.objectify:objectify:jar:6.0.5 -> com.google.cloud:google-cloud-datastore:jar:1.101.0 -> io.grpc:grpc-core:jar:1.24.1 -> io.grpc:grpc-api:jar:[1.24.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.collectDependencies(DefaultDependencyCollector.java:293) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveDependencies(DefaultRepositorySystem.java:307) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	... 60 common frames omitted
Caused by: org.eclipse.aether.resolution.VersionRangeResolutionException: No versions available for io.grpc:grpc-api:jar:[1.24.1] within specified range
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.filterVersions(DefaultDependencyCollector.java:651) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.processDependency(DefaultDependencyCollector.java:396) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.processDependency(DefaultDependencyCollector.java:365) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.process(DefaultDependencyCollector.java:353) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.doRecurse(DefaultDependencyCollector.java:507) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.processDependency(DefaultDependencyCollector.java:460) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.processDependency(DefaultDependencyCollector.java:365) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.process(DefaultDependencyCollector.java:353) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.doRecurse(DefaultDependencyCollector.java:507) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.processDependency(DefaultDependencyCollector.java:460) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.processDependency(DefaultDependencyCollector.java:365) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.process(DefaultDependencyCollector.java:353) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.collectDependencies(DefaultDependencyCollector.java:256) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	... 61 common frames omitted
Tests run: 7, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 4.724 s <<< FAILURE! - in com.google.cloud.tools.eclipse.appengine.libraries.model.CloudLibrariesInPluginXmlTest
testObjectifyLibraryConfig(com.google.cloud.tools.eclipse.appengine.libraries.model.CloudLibrariesInPluginXmlTest)  Time elapsed: 4.557 s  <<< FAILURE!
java.lang.AssertionError: guava not found
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertNotNull(Assert.java:712)
	at com.google.cloud.tools.eclipse.appengine.libraries.model.CloudLibrariesInPluginXmlTest.testObjectifyLibraryConfig(CloudLibrariesInPluginXmlTest.java:163)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray2(ReflectionUtils.java:206)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:161)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:84)
	at org.eclipse.tycho.surefire.osgibooter.OsgiSurefireBooter.run(OsgiSurefireBooter.java:110)
	at org.eclipse.tycho.surefire.osgibooter.HeadlessTestApplication.run(HeadlessTestApplication.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.equinox.internal.app.EclipseAppContainer.callMethodWithException(EclipseAppContainer.java:587)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:198)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:656)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:592)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1498)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1471)

testServletApiLibraryConfig(com.google.cloud.tools.eclipse.appengine.libraries.model.CloudLibrariesInPluginXmlTest)  Time elapsed: 0 s
testObjectify6LibraryConfig(com.google.cloud.tools.eclipse.appengine.libraries.model.CloudLibrariesInPluginXmlTest)  Time elapsed: 0.163 s  <<< FAILURE!
java.lang.AssertionError: 1 dependencies
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.google.cloud.tools.eclipse.appengine.libraries.model.CloudLibrariesInPluginXmlTest.testObjectify6LibraryConfig(CloudLibrariesInPluginXmlTest.java:207)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray2(ReflectionUtils.java:206)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:161)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:84)
	at org.eclipse.tycho.surefire.osgibooter.OsgiSurefireBooter.run(OsgiSurefireBooter.java:110)
	at org.eclipse.tycho.surefire.osgibooter.HeadlessTestApplication.run(HeadlessTestApplication.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.equinox.internal.app.EclipseAppContainer.callMethodWithException(EclipseAppContainer.java:587)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:198)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:656)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:592)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1498)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1471)

testLibrarySize(com.google.cloud.tools.eclipse.appengine.libraries.model.CloudLibrariesInPluginXmlTest)  Time elapsed: 0.002 s
```
</details>

<details>
<summary>LibraryTest.testDirectDependencies</summary>

```
Running com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryTest
00:34:03.973 [main] ERROR c.g.c.t.e.a.libraries.model.Library - Could not load library MavenCoordinates [repository=central, com.googlecode.objectify:objectify:jar:null:6.0.5]
org.eclipse.core.runtime.CoreException: Could not resolve dependencies
	at com.google.cloud.tools.eclipse.util.DependencyResolver._getTransitiveDependencies(DependencyResolver.java:103) ~[com.google.cloud.tools.eclipse.util-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.util.DependencyResolver.lambda$0(DependencyResolver.java:63) ~[com.google.cloud.tools.eclipse.util-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.util.MavenUtils.lambda$1(MavenUtils.java:124) ~[com.google.cloud.tools.eclipse.util-0.1.0-SNAPSHOT.jar:na]
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.executeBare(MavenExecutionContext.java:177) ~[org.eclipse.m2e.core-1.9.0.20180606-2036.jar:na]
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.execute(MavenExecutionContext.java:151) ~[org.eclipse.m2e.core-1.9.0.20180606-2036.jar:na]
	at org.eclipse.m2e.core.internal.embedder.MavenExecutionContext.execute(MavenExecutionContext.java:99) ~[org.eclipse.m2e.core-1.9.0.20180606-2036.jar:na]
	at com.google.cloud.tools.eclipse.util.MavenUtils.runOperation(MavenUtils.java:120) ~[com.google.cloud.tools.eclipse.util-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.util.DependencyResolver.getTransitiveDependencies(DependencyResolver.java:60) ~[com.google.cloud.tools.eclipse.util-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryFactory.loadTransitiveDependencies(LibraryFactory.java:120) ~[com.google.cloud.tools.eclipse.appengine.libraries-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.appengine.libraries.model.Library.resolveDependencies(Library.java:211) [com.google.cloud.tools.eclipse.appengine.libraries-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.appengine.libraries.model.Library.getAllDependencies(Library.java:117) [com.google.cloud.tools.eclipse.appengine.libraries-0.1.0-SNAPSHOT.jar:na]
	at com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryTest.testDirectDependencies(LibraryTest.java:176) [classes/:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_201]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_201]
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50) [junit.jar:4.12]
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12) [junit.jar:4.12]
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47) [junit.jar:4.12]
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17) [junit.jar:4.12]
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325) [junit.jar:4.12]
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78) [junit.jar:4.12]
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57) [junit.jar:4.12]
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290) [junit.jar:4.12]
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71) [junit.jar:4.12]
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288) [junit.jar:4.12]
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58) [junit.jar:4.12]
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268) [junit.jar:4.12]
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363) [junit.jar:4.12]
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365) [org.eclipse.tycho.surefire.junit4-1.3.0.jar:na]
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273) [org.eclipse.tycho.surefire.junit4-1.3.0.jar:na]
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238) [org.eclipse.tycho.surefire.junit4-1.3.0.jar:na]
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159) [org.eclipse.tycho.surefire.junit4-1.3.0.jar:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_201]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_201]
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray2(ReflectionUtils.java:206) [surefire-api-2.22.0.jar:2.22.0]
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:161) [surefire-booter-2.22.0.jar:2.22.0]
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:84) [surefire-booter-2.22.0.jar:2.22.0]
	at org.eclipse.tycho.surefire.osgibooter.OsgiSurefireBooter.run(OsgiSurefireBooter.java:110) [org.eclipse.tycho.surefire.osgibooter-1.3.0.jar:na]
	at org.eclipse.tycho.surefire.osgibooter.HeadlessTestApplication.run(HeadlessTestApplication.java:21) [org.eclipse.tycho.surefire.osgibooter-1.3.0.jar:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_201]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_201]
	at org.eclipse.equinox.internal.app.EclipseAppContainer.callMethodWithException(EclipseAppContainer.java:587) [org.eclipse.equinox.app-1.3.500.v20171221-2204.jar:na]
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:198) [org.eclipse.equinox.app-1.3.500.v20171221-2204.jar:na]
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134) [org.eclipse.osgi-3.13.0.v20180409-1500.jar:na]
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104) [org.eclipse.osgi-3.13.0.v20180409-1500.jar:na]
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388) [org.eclipse.osgi-3.13.0.v20180409-1500.jar:na]
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243) [org.eclipse.osgi-3.13.0.v20180409-1500.jar:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_201]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_201]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_201]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_201]
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:656) [org.eclipse.equinox.launcher-1.5.0.v20180512-1130.jar:na]
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:592) [org.eclipse.equinox.launcher-1.5.0.v20180512-1130.jar:na]
	at org.eclipse.equinox.launcher.Main.run(Main.java:1498) [org.eclipse.equinox.launcher-1.5.0.v20180512-1130.jar:na]
	at org.eclipse.equinox.launcher.Main.main(Main.java:1471) [org.eclipse.equinox.launcher-1.5.0.v20180512-1130.jar:na]
Caused by: org.eclipse.aether.resolution.DependencyResolutionException: Failed to collect dependencies at com.googlecode.objectify:objectify:jar:6.0.5 -> com.google.cloud:google-cloud-datastore:jar:1.101.0 -> io.grpc:grpc-core:jar:1.24.1 -> io.grpc:grpc-api:jar:[1.24.1]
	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveDependencies(DefaultRepositorySystem.java:351) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at com.google.cloud.tools.eclipse.util.DependencyResolver._getTransitiveDependencies(DependencyResolver.java:92) ~[com.google.cloud.tools.eclipse.util-0.1.0-SNAPSHOT.jar:na]
	... 59 common frames omitted
Caused by: org.eclipse.aether.collection.DependencyCollectionException: Failed to collect dependencies at com.googlecode.objectify:objectify:jar:6.0.5 -> com.google.cloud:google-cloud-datastore:jar:1.101.0 -> io.grpc:grpc-core:jar:1.24.1 -> io.grpc:grpc-api:jar:[1.24.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.collectDependencies(DefaultDependencyCollector.java:293) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveDependencies(DefaultRepositorySystem.java:307) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	... 60 common frames omitted
Caused by: org.eclipse.aether.resolution.VersionRangeResolutionException: No versions available for io.grpc:grpc-api:jar:[1.24.1] within specified range
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.filterVersions(DefaultDependencyCollector.java:651) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.processDependency(DefaultDependencyCollector.java:396) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.processDependency(DefaultDependencyCollector.java:365) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.process(DefaultDependencyCollector.java:353) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.doRecurse(DefaultDependencyCollector.java:507) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.processDependency(DefaultDependencyCollector.java:460) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.processDependency(DefaultDependencyCollector.java:365) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.process(DefaultDependencyCollector.java:353) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.doRecurse(DefaultDependencyCollector.java:507) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.processDependency(DefaultDependencyCollector.java:460) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.processDependency(DefaultDependencyCollector.java:365) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.process(DefaultDependencyCollector.java:353) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	at org.eclipse.aether.internal.impl.DefaultDependencyCollector.collectDependencies(DefaultDependencyCollector.java:256) ~[maven-resolver-impl-1.1.1.jar:1.1.1]
	... 61 common frames omitted
Tests run: 26, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.189 s <<< FAILURE! - in com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryTest
testSetJavaVersion(com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryTest)  Time elapsed: 0 s
testTransportDefaultsToHttp(com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryTest)  Time elapsed: 0 s
testToString(com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryTest)  Time elapsed: 0 s
testLibraryFilesDefaultsToEmpty(com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryTest)  Time elapsed: 0 s
testExportDefaultsToTrue(com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryTest)  Time elapsed: 0 s
testSetRelease(com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryTest)  Time elapsed: 0 s
testResolvedDuplicates(com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryTest)  Time elapsed: 0 s
testConstructorNullArgument(com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryTest)  Time elapsed: 0 s
testStageDefaultsToGA(com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryTest)  Time elapsed: 0 s
testSetNullName(com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryTest)  Time elapsed: 0 s
testSetLibraryDependencies(com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryTest)  Time elapsed: 0 s
testSetLibraryDependencies_null(com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryTest)  Time elapsed: 0 s
testDirectDependencies(com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryTest)  Time elapsed: 0.092 s  <<< FAILURE!
java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryTest.testDirectDependencies(LibraryTest.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray2(ReflectionUtils.java:206)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:161)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:84)
	at org.eclipse.tycho.surefire.osgibooter.OsgiSurefireBooter.run(OsgiSurefireBooter.java:110)
	at org.eclipse.tycho.surefire.osgibooter.HeadlessTestApplication.run(HeadlessTestApplication.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.equinox.internal.app.EclipseAppContainer.callMethodWithException(EclipseAppContainer.java:587)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:198)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:656)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:592)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1498)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1471)

testSetSiteUri(com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryTest)  Time elapsed: 0 s
```
</details>

--- a/plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibrariesInPluginXmlTest.java
+++ b/plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibrariesInPluginXmlTest.java
@@ -80,7 +80,7 @@ public class CloudLibrariesInPluginXmlTest {
     assertThat(mavenCoordinates.getArtifactId(), is("appengine-api-1.0-sdk"));
     
     DefaultArtifactVersion actual = new DefaultArtifactVersion(mavenCoordinates.getVersion());
-    DefaultArtifactVersion expected = new DefaultArtifactVersion("1.9.74");
+    DefaultArtifactVersion expected = new DefaultArtifactVersion("1.9.77");
     assertTrue(actual.compareTo(expected) >= 0);
     
     assertThat(mavenCoordinates.getType(), is("jar"));
@@ -168,7 +168,7 @@ public class CloudLibrariesInPluginXmlTest {
     assertThat(objectifyMavenCoordinates.getArtifactId(), is("objectify"));
     DefaultArtifactVersion artifactVersion = new DefaultArtifactVersion(
         objectifyMavenCoordinates.getVersion());
-    assertEquals(new DefaultArtifactVersion("6.0.4"), artifactVersion);
+    assertEquals(new DefaultArtifactVersion("6.0.5"), artifactVersion);
     assertThat(objectifyMavenCoordinates.getType(), is("jar"));
     assertNull(objectifyMavenCoordinates.getClassifier());
 
@@ -184,7 +184,7 @@ public class CloudLibrariesInPluginXmlTest {
     assertThat(guavaMavenCoordinates.getRepository(), is("central"));
     assertThat(guavaMavenCoordinates.getGroupId(), is("com.google.guava"));
     assertThat(guavaMavenCoordinates.getArtifactId(), is("guava"));
-    assertThat(guavaMavenCoordinates.getVersion(), is("27.1-android"));
+    assertThat(guavaMavenCoordinates.getVersion(), is("28.1-android"));
     assertThat(guavaMavenCoordinates.getType(), is("jar"));
     assertNull(guavaMavenCoordinates.getClassifier());
 
@@ -242,7 +242,7 @@ public class CloudLibrariesInPluginXmlTest {
     assertThat(guavaMavenCoordinates.getRepository(), is("central"));
     assertThat(guavaMavenCoordinates.getGroupId(), is("com.google.guava"));
     assertThat(guavaMavenCoordinates.getArtifactId(), is("guava"));
-    assertThat(guavaMavenCoordinates.getVersion(), is("27.1-android"));
+    assertThat(guavaMavenCoordinates.getVersion(), is("28.1-android"));
     assertThat(guavaMavenCoordinates.getType(), is("jar"));
     assertNull(guavaMavenCoordinates.getClassifier());
 

--- a/plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml
+++ b/plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml
@@ -18,7 +18,7 @@
         <mavenCoordinates
               artifactId="appengine-api-1.0-sdk"
               groupId="com.google.appengine" 
-              version="1.9.74" />
+              version="1.9.77" />
         <exclusionFilter pattern="com/google/appengine/repackaged/**" />
         <exclusionFilter pattern="com/google/appengine/labs/repackaged/**" />
         <inclusionFilter pattern="com/google/apphosting/api/**" />
@@ -53,11 +53,11 @@
           siteUri="https://github.com/objectify/objectify/wiki" >
       <libraryFile
           export="true"
-          javadocUri="https://www.javadoc.io/doc/com.googlecode.objectify/objectify/6.0.3">
+          javadocUri="https://www.javadoc.io/doc/com.googlecode.objectify/objectify/6.0.5">
         <mavenCoordinates
               artifactId="objectify"
               groupId="com.googlecode.objectify"
-              version="6.0.4" />
+              version="6.0.5" />
       </libraryFile>
     </library>
     
@@ -71,11 +71,11 @@
           siteUri="https://github.com/objectify/objectify/wiki" >
       <libraryFile
           export="true"
-          javadocUri="https://www.javadoc.io/doc/com.googlecode.objectify/objectify/6.0.3">
+          javadocUri="https://www.javadoc.io/doc/com.googlecode.objectify/objectify/6.0.5">
         <mavenCoordinates
               artifactId="objectify"
               groupId="com.googlecode.objectify"
-              version="6.0.4" />
+              version="6.0.5" />
       </libraryFile>
     </library>
 

